pages/getting-started/cloud-shell-quickstart.md

```diff
@@ -24,11 +24,76 @@ Navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Clo
 
 ### Set up a Cloud Provider
 
-Choose the option to set up your Cloud Provider in the browser. Plural can be set up on AWS, GCP, or Azure.
+Choose the option to continue your setup in the browser. If you'd prefer to use the CLI, choose "Local terminal" and switch to the CLI quickstart.
 
 ![](</assets/cloud-shell-quickstart/setup-options.png>)
 
-You can provide your credentials in the following screen.
+Plural can be set up on AWS, GCP, or Azure. You can provide your credentials in the following screen:
+
+![](</assets/cloud-shell-quickstart/cloud-provider-info.png>)
+
+Each cloud provider requires a slightly different set of credentials. Follow the steps below to generate the credentials for your provider.
+
+{% tabs %}
+{% tab title="AWS" %}
+For AWS, you'll need to create or use a user with AdministratorAccess and create an access key for Plural to use.
+
+To create a new user, navigate to the IAM section of your AWS Console.
+
+![](</assets/cloud-shell-quickstart/iam-aws.png>)
+
+Select the `Users` tab and `Add users`. You should see the following screen:
```

Comment:
I think the word "click" might be missing before "Add users"?

Reply:
addressed